packages/edit-site/src/components/navigation-sidebar/navigation-panel/template-hierarchy.js

```diff
@@ -21,3 +35,49 @@ export function isTemplateSuperseded( slug, existingSlugs, showOnFront ) {
 			isTemplateSuperseded( overrideSlug, existingSlugs, showOnFront )
 	);
 }
+
+export function getTemplateLocation( slug ) {
+	const isTopLevelTemplate = TEMPLATES_TOP_LEVEL.includes( slug );
+	if ( isTopLevelTemplate ) {
+		return MENU_TEMPLATES;
+	}
+
+	const isGeneralTemplate = TEMPLATES_GENERAL.includes( slug );
+	if ( isGeneralTemplate ) {
+		return MENU_TEMPLATES_GENERAL;
+	}
+
+	const isPostsTemplate = TEMPLATES_POSTS_PREFIXES.some( ( prefix ) =>
+		slug.startsWith( prefix )
+	);
+	if ( isPostsTemplate ) {
+		return MENU_TEMPLATES_POSTS;
+	}
+
+	const isPagesTemplate = TEMPLATES_PAGES_PREFIXES.some( ( prefix ) =>
+		slug.startsWith( prefix )
+	);
+	if ( isPagesTemplate ) {
+		return MENU_TEMPLATES_PAGES;
+	}
+
+	return MENU_TEMPLATES_GENERAL;
+}
+
+export function getUnusedTemplates( templates, showOnFront ) {
+	const unusedTemplates = [];
+
+	const templateSlugs = map( templates, 'slug' );
+	const supersededTemplates = templates.filter( ( { slug } ) =>
+		isTemplateSuperseded( slug, templateSlugs, showOnFront )
+	);
+
+	return [ ...supersededTemplates, ...unusedTemplates ];
```

Comment:
I noticed we aren't using the `unusedTemplates` here. Can we just return `supersededTemplates`?